markdown/org/docs/sewing/knit-binding/en.md

```diff
@@ -79,41 +80,50 @@ Stretch your binding too much, and it will spring back too far, causing the main
+On a slight curve, the difference between the inner and outer edge is not so big, and a little bit of stretch is all you need. But on a tighter curve, the difference between the inner and outer edge of your binding will be more significant. And, you'll need more stretch to accommodate for that.
 
-So as you follow your seam, and it curves around necklines, or shoulders, or design features, you will have to adapt the amount of stretch to the situation.
+So as you follow your seam and as it curves around necklines, shoulders, or design features, you will have to adapt the amount of stretch to the situation.
 
 ## You cannot calculate the length of your knit binding
 
-I am not saying it is theoretically impossible. But I am saying that it is practically impossible.
+It may not be theoretically impossible, but it is practically impossible.
 
-The length of your knit binding depends on the amount it needs to be stretched. This depends on the curves you have to follow, and the width of the binding, not to mention the strength of its stretchiness. There's just too many parameters to take into account to calculate your binding length accurately.
+The length of your knit binding depends on the amount it needs to be stretched. This depends on the curves you have to follow and the width of the binding, not to mention the strength of its stretchiness. There are just too many parameters to take into account to calculate your binding length accurately.
 
 ### Do not do this
 
 There are a bunch of tutorials out on the internet that show you how to sew on knit binding.
 Many of those suggest something like _make the neck binding 90% of the length of the seam you are attaching it to_ (the percentage varies).
 
-This idea is that you cut your binding, join its edges, and then sew it in the opening. That is (in my opinion) no good.
+This idea is that you would cut your binding, join its edges to form a closed loop, and then sew it to the opening. However, this is not the best method.
 
-An alternative approach to sidestep the length issue is to attach the binding before you close the opening your binding goes in.
-For example, on a T-shirt you close one shoulder seam, attach the binding, then close the second shoulder seam.
+A possible alternate approach to sidestep the length issue would be to attach the binding before you close the opening your binding goes in.
+For example, on a T-shirt you would close one shoulder seam, attach the binding, then close the second shoulder seam.
 
-Obviously, that means your shoulder seam will cut through your binding which is a big no no in my book.
+However, that obviously means your shoulder seam would cut through your binding.
+So, this approach is also not desirable.
```

Comment:
The same issue as earlier, this again changes the way the documentation lands with the reader.